DOCS/man/options.rst

```diff
@@ -4138,19 +4138,21 @@ Demuxer
     opens the URL of the next playlist entry as soon the current URL is fully
     read.
 
+    This is most useful when reading small files where a significant portion of
+    the load time is spent creating a new demuxer thread, such as images, which
+    get fully prefetched.
+
     This does **not** work with URLs resolved by the ``youtube-dl`` wrapper,
     and it won't.
 
-    This can give subtly wrong results if per-file options are used, or if
-    options are changed in the time window between prefetching start and next
-    file played.
+    If ``--demuxer`` and related probing options are changed in the time window
+    between the start of prefetching and the playback of the next file, the old
+    values are used.
```

Comment:
So it breaks all format specific options and profiles? Kind of defeats the purpose of extension profiles, no?

Reply:
Well that's not purpose of extension profiles, they were what was used before implementing conditional profiles and then were deprecated. In conditional profiles this already doesn't work because they are evaluted too late, it seems to work for the first file only if you match by `path`, and this still works with `--prefetch-playlist` which doesn't affect the first file.

But yes, setting demuxer options in extension profiles is a reason for not enabling it by default globally. I think it is worth the trade-off for images because you navigate between then much more, and yes the next file could be a video, defaults can't be perfect, but navigating between images faster should be preferable for the majority of users. A note about it can be added in the image profile.